src/error-handler.ts

```diff
@@ -0,0 +1,14 @@
+import { ErrorRequestHandler } from 'express';
+import Logger from './modules/logger/logger';
+
+const serverErrorHandler: ErrorRequestHandler = (err, _req, res, _next) => {
+  const errorCode = err.code as string;
+  const errorMessage = err.message as string;
+  Logger.error(`Error: ${errorCode} ${errorMessage}`);
+  const errorObj = {
+    message: errorMessage,
+  };
+  res.status(err.statusCode).json(errorObj);
```

Comment:
Do you want to leave the serialization of error to individual error class? For example, when a user submits a form, a module may want to include field wise validation error.

Maybe each class should do `toJson`?

Open to other ideas